extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -263,4 +291,36 @@ declare module 'sqldbproj' {
 		sqlDW = 'Azure Synapse Dedicated SQL Pool',
 		sqlEdge = 'Azure SQL Edge'
 	}
+	export interface ISqlConnectionProperties {
```

Comment:
Let me know if you have doc descriptions I could give these, it's not as important for extension APIs but it's still generally nice to have.